Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "manta-signer"
 edition = "2021"
-version = "1.1.1"
+version = "1.1.2"
```

Comment:
If following semantic versioning it probably should at least be 1.2.0.

Ok as is if binaries are already set to go

Reply:
Normally we would chnage the second number with very big changes while the first would be large refactoring.